File: src/assets-en/courses/db-msc/gyak05/LINQ/Model/Person.cs
namespace LINQ.Model
{
    public class Person
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public decimal Salary { get; set; }

        public override string ToString()
        {
            return string.Format("Person[FirstName = {0}, LastName = {1}, Salary = {2}]", FirstName, LastName, Salary);
        }
    }
}

File: src/assets-source/courses/xml/lab07/solution/src/main/java/com/github/szgabsz91/iitszg/xml/lab07/exercise01/model/Person.java
package com.github.szgabsz91.iitszg.xml.lab07.exercise01.model;

import jakarta.xml.bind.annotation.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class Person {

    @XmlAttribute
    private int id;
    @XmlElement(name = "first-name")
    private String firstName;
    @XmlElement(name = "last-name")
    private String lastName;

}
